inc/wordpress-shims.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * WordPress shims.
+ *
+ * @package storefront
+ */
+
+if ( ! function_exists( 'wp_body_open' ) ) {
+    /**
+     * Adds backwards compatibility for wp_body_open() introduced with WordPress 5.2
+     *
+     * @since 2.5.3
```

Comment:
A release was done after this PR was created, so we will need to update this line to `2.5.4`.
```suggestion
     * @since 2.5.4
```

Reply:
@nielslange can you apply this change on your branch? Then we can merge this :)

Reply:
@haszari The version number had been adjusted according to your feedback. 😀